worker/test/src/RTC/Codecs/TestVP9.cpp

```diff
@@ -75,4 +75,111 @@ SCENARIO("process VP9 payload descriptor", "[codecs][vp9]")
 		forwarded = ProcessVP9Packet(context, 1, 1);
 		REQUIRE_FALSE(forwarded);
 	}
+
+	SECTION("test PayloadDescriptorHandler")
+	{
+		RTC::Codecs::EncodingContext::Params params;
+		params.spatialLayers  = 1;
+		params.temporalLayers = 3;
+
+		Codecs::VP9::EncodingContext context(params);
+
+		uint16_t start = MaxPictureId - 2000;
+
+		context.SetCurrentTemporalLayer(0, start + 0);
+		context.SetCurrentTemporalLayer(1, start + 1);
+		context.SetCurrentTemporalLayer(2, start + 5);
+		context.SetCurrentTemporalLayer(0, start + 6);
+
+		REQUIRE(context.GetTemporalLayerForPictureId(start + 0) == 0);
+		REQUIRE(context.GetTemporalLayerForPictureId(start + 1) == 1);
+		REQUIRE(context.GetTemporalLayerForPictureId(start + 2) == 1);
+		REQUIRE(context.GetTemporalLayerForPictureId(start + 5) == 2);
+		REQUIRE(context.GetTemporalLayerForPictureId(start + 6) == 0);
+
+		context.SetCurrentTemporalLayer(1, start + 1000);
+		context.SetCurrentTemporalLayer(2, start + 1001); // This will drop the first item.
+
+		REQUIRE(context.GetTemporalLayerForPictureId(start + 1000) == 1);
+		REQUIRE(context.GetTemporalLayerForPictureId(start + 0) == 1); // It will get the item at start+1.
+
+		context.SetCurrentTemporalLayer(0, 0); // This will drop items from start to start+999.
+
+		REQUIRE(context.GetTemporalLayerForPictureId(0) == 0);
+		REQUIRE(
+		  context.GetTemporalLayerForPictureId(start + 0) == 1); // It will get the item at start+1000.
+	}
+
+	SECTION("drop packets that belong to other temporal layers with unordered pictureID")
+	{
+		RTC::Codecs::EncodingContext::Params params;
+		params.spatialLayers  = 1;
+		params.temporalLayers = 3;
+
+		Codecs::VP9::EncodingContext context(params);
+		context.SyncRequired();
+		context.SetCurrentSpatialLayer(0, 0);
+		context.SetTargetSpatialLayer(0);
+
+		uint16_t start                                                     = MaxPictureId - 20;
+		std::vector<std::tuple<uint16_t, uint16_t, int16_t, bool>> packets = {
+			// targetTemporalLayer=0
+			{ start, 0, 0, true },
+			{ start, 1, -1, false },
+			{ start + 1, 0, -1, true },
+			{ start + 1, 1, -1, false },
+			{ start + 2, 0, -1, true },
+			{ start + 2, 1, -1, false },
+			// targetTemporalLayer=1
+			{ start + 10, 0, 1, true },
+			{ start + 10, 1, -1, true },
+			{ start + 11, 0, -1, true },
+			{ start + 11, 1, -1, true },
+
+			{ start + 3, 0, -1, true }, // old packet
```

Comment:
This blank line doesn't make sense IMHO.
```suggestion
			{ start + 11, 1, -1, true },
			{ start + 3, 0, -1, true }, // old packet
```